src/FxCop/System.Globalization.Analyzers/Test/Globalization/CA1304SpecifyCultureInfoTests.cs

```diff
@@ -33,10 +33,35 @@ void M(string strA, string strB)
 }",
             GetCA1304CSharpDefaultResultAt(9, 25, callee: "string.ToLower()", caller: "C.M(string, string)", preferred: "string.ToLower(System.Globalization.CultureInfo)"),
             GetCA1304CSharpDefaultResultAt(10, 25, callee: "string.ToUpper()", caller: "C.M(string, string)", preferred: "string.ToUpper(System.Globalization.CultureInfo)"));
-        } 
-        
+
+            VerifyBasic(@"
+Imports System
+Imports System.Globalization
+
+Public Module C
+    Public Sub M(strA As String, strB As String)
+        Dim strAld As String = strA.ToLower()
+        Dim strBud As String = strB.ToUpper()
+
+        ' The following calls are okay as far as CA1304 is concerned
+        Dim strAlc As String = strA.ToLower(CultureInfo.CurrentCulture)
+        Dim strBuc As String = strB.ToUpper(CultureInfo.InvariantCulture)
+        If ((strAld = strAlc) <> (strBud = strBuc)) Then
+            Console.WriteLine(""Oops"")
+        End If
+    End Sub
+End Module",
+            GetCA1304BasicDefaultResultAt(7, 32, callee: "Public Overloads Function ToLower() As String",
+                                                 caller: "Public Sub M(strA As String, strB As String)",
+                                                 preferred: "Public Overloads Function ToLower(culture As System.Globalization.CultureInfo) As String"),
+            GetCA1304BasicDefaultResultAt(8, 32, callee: "Public Overloads Function ToUpper() As String",
+                                                 caller: "Public Sub M(strA As String, strB As String)",
+                                                 preferred: "Public Overloads Function ToUpper(culture As System.Globalization.CultureInfo) As String"));
+
+        }
+
         [Fact]
-        public void CA1304ShoudUseOverloadsWithExplicitCultureInfoParameterTests_Misc_CS()
+        public void CA1304CultureInfoParameterTests_Misc()
         {
             VerifyCSharp(@"
 using System;
```

Comment:
Make sure you expand the code during review. Line 67 is "using System;". Lines 68-71 are other using statements.